examples/gradient_descent_s2.py

```diff
@@ -18,11 +18,14 @@
 import numpy as np
 
 from geomstats.hypersphere import Hypersphere
+from geomstats.spd_matrices_space import SPDMatricesSpace
 import geomstats.vectorization as vectorization
 import geomstats.visualization as visualization
 
+
 SPHERE2 = Hypersphere(dimension=2)
 METRIC = SPHERE2.metric
+np.random.seed(1984)
 
 
 def gradient_descent(start,
```

Comment:
Just a thought. This works for any manifold embedded in a euclidean space. Later we should probably divide geomstats' code into: 'geometry', 'statistics', 'visualization', 'optimization', 'tests' folders.